### PR TITLE
Factorize the 3 file names used in docker-app

### PR DESCRIPTION
**Pull request by silvin-lubecki**
**- A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

File: internal/names.go [hunk 1]
```diff
@@ -14,6 +14,11 @@ const (
 	ImageLabel = "com.docker.application"
 )
 
+var (
+	// Application file names, in order.
+	FileNames = []string{"metadata.yml", "docker-compose.yml", "settings.yml"}
```
Comment (vdemeester) [inline internal/names.go]: Should we put those names into constants ? and that thing in a constant too :angel: (so that we can't change it :stuck_out_tongue_closed_eyes:)

Comment (silvin-lubecki) [reply internal/names.go]: Can't put a slice as a constant sorry...

Comment (vdemeester) [reply internal/names.go]: @silvin-lubecki right, but the value inside the slice could :stuck_out_tongue_closed_eyes:

File: internal/types/metadata.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ type AppMetadata struct {
 	Version     string
 	Name        string
 	Description string
-	Namespace   string `yaml:"namespace"`
```
Comment (mat007) [inline internal/types/metadata.go]: Was this change intended? In the same commit?

Comment (silvin-lubecki) [reply internal/types/metadata.go]: Yeah right it's in the same commit... It's just something I spotted. But there's a comment on it in the commit message 😄

File: internal/names.go (hunk 1 above)
Comment (silvin-lubecki) [reply internal/names.go]: 👍

**Comment by silvin-lubecki** [comment]
@vdemeester PTAL
